modules/contact-form/grunion-contact-form.php

```diff
@@ -1542,7 +1542,20 @@ static function parse( $attributes, $content ) {
 			$r .= "<form action='" . esc_url( $url ) . "' method='post' class='contact-form commentsblock'>\n";
 			$r .= $form->body;
 			$r .= "\t<p class='contact-submit'>\n";
-			$r .= "\t\t<input type='submit' value='" . esc_attr( $form->get_attribute( 'submit_button_text' ) ) . "' class='pushbutton-wide'/>\n";
+			
+			$submit_button_class = 'pushbutton-wide';
+			
+			/**
+			 * Filter the contact form submit button class attribute.
+			 *
+			 * @module contact-form
+			 * @since 5.3.0
```

Comment:
Some time has passed since this PR has been submitted, so please edit this to be `5.5.0` instead - this will only make it in the shipped code in 5.5.0.